src/main/java/seedu/address/model/module/ModuleName.java

```diff
@@ -10,13 +10,13 @@
 public class ModuleName {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Module names should only contain alphanumeric characters, and it should not be blank";
```

Comment:
I like the improvement to readability made by using Module names instead of Names to differentiate from other types of names.

Reply:
Thank you 👍